DBCON-279: add docs for db connector minor release 1.10.0 (#1497)
mulesoft/docs-connectors: db/1.10/modules/ROOT/pages/database-connector-connection.adoc

* In the *Database Config* window, click *Configure*, select *Add Maven dependency*, and pick the dependency:
+
.Pick Maven dependency
image::database-dependency.png[Pick Maven dependency]
+
* Add the dependency directly to your project's `pom.xml` file:
+
[source,xml,linenums]
----
<dependency>
    <groupId>mysql</groupId>
    <artifactId>mysql-connector-java</artifactId>
    <version>5.1.48</version>
</dependency>
----
+
. Configure your `pom.xml` file so that Mule exposes the driver dependency to Database Connector:
+
[source,xml,linenums]
----
<build>
    <plugins>
        <!-- Only used to declare the shared libraries-->
        <plugin>
            <groupId>org.mule.tools.maven</groupId>
            <artifactId>mule-maven-plugin</artifactId>
            <version>1.0.0</version>
            <configuration>
                <sharedLibraries>
                    <sharedLibrary>
                        <groupId>mysql</groupId>
                        <artifactId>mysql-connector-java</artifactId>
                    </sharedLibrary>
                </sharedLibraries>
            </configuration>
        </plugin>
    </plugins>
</build>
----

== Configure Connection Pools

> Establishing connections is costly in terms of connectors, relational database management system (RDBMS) network, and processing overhead. Database Connector supports the use of pooling profiles that enable you to build a connection pool. This is a group of established connections that the connector can use to perform required operations without having to establish new connections every time.

Reviewer: ```suggestion
Establishing connections is costly in terms of connectors, the relational database management system (RDBMS) network, and processing overhead. Database Connector supports the use of pooling profiles that enable you to build a connection pool. This is a group of established connections that the connector can use to perform required operations without having to establish new connections every time.
```

> When you configure the *Pooling profile* field in the global element of your application, deselect the *Test connection on checkout* field to improve the connector performance with connection pooling. This is because if you leave the field selected as default (`TRUE`), every time a connection is checked out of the pool the connection pool performs connectivity checks against the database which increase the time required for each operation to complete.

Reviewer: ```suggestion
When you configure the *Pooling profile* field in the global element of your application, deselect the *Test connection on checkout* field to improve the connector performance with connection pooling. If you leave the field selected as default (`TRUE`), every time a connection is checked out of the pool, the connection pool performs connectivity checks against the database, which increases the time required for each operation to complete.
```

All connection type configurations, except the *Data Source Reference Connection*, accept the *Pooling profile* field configuration. In JDBC, pooling occurs at the data source level, so to enable pooling, you have to configure the global data source.

> The following example shows how to configure a pooling profile for a MySQL database connection and the *Test connection on checkout* field to be `FALSE`:

Reviewer: ```suggestion
The following example shows how to configure a pooling profile for a MySQL database connection with the *Test connection on checkout* field set to `FALSE`:
```

. In the *Database Config* window, in the *Advanced* tab, for *Pooling profile* select *Edit inline*.
. Set the following fields:

> * *Max pool size*: `30` +

Reviewer: I don't think you need the + at the end of each to these single list items. If it doesn't show up in the output, don't worry about it, but I think it might show up.
